ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -236,39 +240,28 @@ export default function CreatePassword({
 
   return (
     <div className="create-password__wrapper" data-testid="create-password">
-      {
-        ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
-        secretRecoveryPhrase &&
-        firstTimeFlowType === FIRST_TIME_FLOW_TYPES.IMPORT ? (
-          <TwoStepProgressBar
-            stage={twoStepStages.PASSWORD_CREATE}
-            marginBottom={4}
-          />
-        ) : (
-          <ThreeStepProgressBar
-            stage={threeStepStages.PASSWORD_CREATE}
-            marginBottom={4}
-          />
-        )
-        ///: END:ONLY_INCLUDE_IN
-      }
-
-      <Typography variant={TypographyVariant.H2} fontWeight={FONT_WEIGHT.BOLD}>
+      {secretRecoveryPhrase &&
+      firstTimeFlowType === FIRST_TIME_FLOW_TYPES.IMPORT ? (
+        <TwoStepProgressBar
+          stage={twoStepStages.PASSWORD_CREATE}
+          marginBottom={4}
+        />
+      ) : (
+        <ThreeStepProgressBar
+          stage={threeStepStages.PASSWORD_CREATE}
+          marginBottom={4}
+        />
+      )}
+      <Text variant={TextVariant.headingLg} marginBottom={3}>
         {t('createPassword')}
-      </Typography>
-
-      <Typography variant={TypographyVariant.H4} align={TEXT_ALIGN.CENTER}>
-        {
-          ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
-          t('passwordSetupDetails')
-          ///: END:ONLY_INCLUDE_IN
-        }
-        {
-          ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
-          t('mmiPasswordSetupDetails')
-          ///: END:ONLY_INCLUDE_IN
-        }
-      </Typography>
+      </Text>
+      <Text
+        variant={TextVariant.headingSm}
+        textAlign={TextAlign.Center}
+        fontWeight={FontWeight.Normal}
+      >
+        {t('passwordSetupDetails')}
```

Comment:
There appears to be some more missing build type comment logic missing
```suggestion
        {
          ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
          t('passwordSetupDetails')
          ///: END:ONLY_INCLUDE_IN
        }
        {
          ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
          t('mmiPasswordSetupDetails')
          ///: END:ONLY_INCLUDE_IN
        }
```